email_account_validity/_store.py

```diff
@@ -44,13 +55,30 @@ def create_table_txn(txn: LoggingTransaction):
                     user_id TEXT PRIMARY KEY,
                     expiration_ts_ms BIGINT NOT NULL,
                     email_sent BOOLEAN NOT NULL,
-                    renewal_token TEXT,
+                    long_renewal_token TEXT,
+                    short_renewal_token TEXT,
```

Comment:
some comments somewhere around here about the difference between `long_renewal_token` and `short_renewal_token` would be good.